src/psyclone/nemo.py

```diff
@@ -71,80 +71,26 @@
 NEMO_INDEX_ORDERING = ["lon", "lat", "levels", "tracers"]
 
 
-class ASTProcessor(object):
+class NemoASTProcessor(fparser2ASTProcessor):
```

Comment:
Do you think ASTProcessor class should be a file of its own instead of psyGen.py? I notice pylint complains that the files are to long and the ASTProcessor and its tests are growing in my following PRs.

Reply:
Hi @sergisiso. I suggest you don't worry about file size in this PR. You are right that we do need to restructure, probably creating a base/common directory which contains files split out from psyGen.py, as well as putting api-specific code in separate directories, but that can wait.